tests/src/tools/ReadyToRun.SuperIlc/ProcessRunner.cs

```diff
@@ -216,17 +215,19 @@ private void StopProcessAtomic()
 
     private void StandardOutputEventHandler(object sender, DataReceivedEventArgs eventArgs)
     {
-        if (!string.IsNullOrEmpty(eventArgs.Data))
+        string data = eventArgs?.Data;
+        if (!string.IsNullOrEmpty(data))
         {
-            _logWriter.WriteLine(eventArgs.Data);
+            _logWriter?.WriteLine(data);
```

Comment:
Is the `?` necessary here? We instantiate `_logWriter` in the constructor (IMO it's better to not silently fail to log and just get the null ref exception)

Reply:
Good point, most likely not; will fix.

Reply:
Fixed in 6th commit.